optimum/onnxruntime/configuration.py

```diff
@@ -620,19 +620,19 @@ class OptimizationConfig:
             The optimized graph might contain operators for GPU or CPU only when `optimization_level` > 1.
         fp16 (`bool`, defaults to `False`):
             Whether all weights and nodes should be converted from float32 to float16.
-        optimize_with_onnxruntime_only (`bool`, defaults to `False`):
-            Whether to only use ONNX Runtime to optimize the model and no graph fusion in Python.
-        disable_gelu (`bool`, defaults to `False`):
+        fuse_operators (`bool`, defaults to `True`):
+            Whether to fuse operators in addition to ONNX Runtime graph optimizations.
+        disable_gelu_fusion (`bool`, defaults to `False`):
             Whether to disable the Gelu fusion.
-        disable_layer_norm (`bool`, defaults to `False`):
+        disable_layer_norm_fusion (`bool`, defaults to `False`):
             Whether to disable Layer Normalization fusion.
-        disable_attention (`bool`, defaults to `False`):
```

Comment:
Just kindly mention that changing the args name might be a problem for users who are already using these flags, like this https://github.com/huggingface/optimum/issues/325#issuecomment-1202467773

Reply:
Yes I agree with you @JingyaHuang. Maybe I can add a warning in the `__post_init__` of the configuration? Or should we just keep the documentation update? What do you think? @echarlaix @michaelbenayoun

Reply:
I think adding a warning in the `__post_init__` will not change much as an error will be thrown when users will try to instantiate an `OptimizationConfig` with the old argument names (before the warning). But maybe you meant allowing users to use the old argument names by modifying the `__init__` and add a warning in the `__post_init__` in order to smoothen this transition ?

Reply:
Yes, you're right. I was thinking about something like that even though I don't like it that much because we would also need to make sure that the value of an argument does not contradict the value of its former version. But I'm not sure if there is a better solution. Or we just keep the current names and make sure it's clear for users that it only fuses operators.

Reply:
I think what can be done is this:

1. We allow both
2. If `difsable_attention` is set, it overwrites the other with a warning saying it overwrites the values, and that it will be deprecated in 3-4 releases
3. Remove the old one in 3-4 releases

What do you think?

Reply:
Sounds good to me :+1: